### PR TITLE
Update the Dashboard and Order-Form Journey

### PR DESCRIPTION
**Pull request by kiran-mastek**
Revert to Primary Organisation if no ODS Code is Included
Skip tests to be fixed by upcoming changes

Story: [AB#14286](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/14286)
Task: [AB#14361](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/14361)

### CODE REVIEW

**Comment by bjss-chris** [comment]
@kiran-mastek, I'll review this as soon as you've resolved the conflicts.

File: app/routes.js [hunk 1]
```diff
@@ -45,6 +46,37 @@ export const routes = (authProvider, sessionManager) => {
     stream.on('close', () => res.end());
   }));
 
+  const regExp = new RegExp('^/organisation/[A-Za-z]\\d{6}-\\d{2}');
+
+  // eslint-disable-next-line consistent-return
```
Comment (bjss-chris) [inline app/routes.js:51]: Rather than disable the ESLint warning we can just do `return next();`

File: app/routes.js [hunk 2]
```diff
@@ -45,6 +46,37 @@ export const routes = (authProvider, sessionManager) => {
...
+            return res.redirect(`${config.baseUrl}/organisation/${odsCode}/select`);
+          }
+          const newUrl = req.url.replace('/organisation/', `/organisation/${odsCode}/order/`);
+
+          return res.redirect(`${config.baseUrl}${newUrl}`);
+        }
+      }
+    }
+
+    next();
```
Comment (bjss-chris) [inline app/routes.js:77]: `return next();` as above.

**Review by bjss-chris — CHANGES_REQUESTED** [review]

File: app/routes.js (hunk 2 above)
Comment (kiran-mastek) [reply app/routes.js]: done

File: app/routes.js (hunk 1 above)
Comment (kiran-mastek) [reply app/routes.js]: done

**Review by bjss-chris — APPROVED** [review]

**Pull request merged**